api/plugins/filter/lagoon.py

```diff
@@ -58,6 +70,8 @@ def autogen_route(self, routes, route_pattern):
 
     def filters(self):
         return {
-            "lagoon_environment_input": self.environment_input_type,
-            "lagoon_autogen_route": self.autogen_route,
+            "autogen_route": self.autogen_route,
```

Comment:
Since we're stuck to using the FQCN for filters, I thought I'd get rid of the `lagoon_` prefix.